components/BCRegistriesAccount.vue

```diff
@@ -1,257 +1,230 @@
+
+        <v-card elevation="2" max-width="375px" class="mr-5 mb-8">
+          <v-card-title class="menu-header pr-0 mr-0">
+            <v-row>
+              <v-col cols="7">
+                <span class="menu-item-title">Wills Registry</span>
+              </v-col>
+              <v-col cols="5">
+                <v-row>
+                  <v-col cols="auto" class="menu-title-badge ml-0 mr-4 pt-0">COMING SOON</v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="auto"></v-col>
+                </v-row>
+              </v-col>
+            </v-row>
+          </v-card-title>
+
+          <v-card-text class="menu-item pl-7 pr-7">
+            <p class="ma-0 pt-8 pb-7">
+              <span style="font-weight: bold">Coming soon</span> to BC
+              Registries Account, Wills Registry allows solicitors, notaries,
+              and title search companies to do the following:
+            </p>
+            <ul>
+              <li><span class="menu-list-text">File a wills notice</span></li>
+              <li>
+                <span class="menu-list-text">Search for an existing wills notice</span>
+              </li>
+            </ul>
+            <!-- p class="pt-7 ma-0 pb-6">
+              <a
+                :href="$config.willsURL"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Go to Wills Registry
+              </a>
+              <v-icon class="pl-1" color="primary" dark dense>mdi-open-in-new</v-icon>
+            </p -->
+          </v-card-text>
+        </v-card>
+
+        <v-card elevation="2" max-width="375px" class="mr-5 mb-8">
+          <v-card-title class="menu-header pr-0 mr-0">
+            <v-row>
+              <v-col cols="9">
+                <span class="menu-item-title">Personal Property Registry</span>
+              </v-col>
+              <v-col cols="3">
+                <v-row>
+                  <v-col cols="auto" class="menu-title-badge ml-0 pl-2 mr-3 pt-0">ALPHA</v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="auto"></v-col>
+                </v-row>
+              </v-col>
+            </v-row>
+          </v-card-title>
+
+          <v-card-text class="menu-item pl-7 pr-7">
+            <p class="ma-0 pt-8 pb-7">
+              A private alpha release of the new Personal Property Registry is
+              available to a select group of users. Current functionality
+              includes:
+            </p>
+            <ul>
+              <li>
+                <span class="menu-list-text">Registration Search</span>
+              </li>
+              <li>
+                <span class="menu-list-text">Download PDF search reports</span>
+              </li>
+            </ul>
+            <p class="pt-7 ma-0 pb-6">
+              <a
+                href="/ppr-marketing"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
```

Comment:
I changed this from a route to a link so it opens in a new tab (as per the icon beside it).